bootstrap.d/managarm-system.y4.yml

```diff
@@ -124,6 +124,52 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: managarm-kernel-uefi
+    metadata:
+      summary: eir for UEFI
+      description: This package provides eir-uefi.
+      spdx: 'MIT'
+      website: 'https://managarm.org'
+      maintainer: "Marvin Friedrich <contact@marvinf.com>"
+      categories: ['sys-kernel']
+    architecture: '@OPTION:arch@'
+    default: true
+    from_source: managarm
+    tools_required:
+      - host-llvm-toolchain
+      - host-managarm-tools
+      - host-pkg-config
+      - virtual: pkgconfig-for-target
+        triple: "@OPTION:arch-triple@"
+    pkgs_required:
+      - frigg
+      - mlibc-headers
+      - freestnd-c-hdrs
+      - freestnd-cxx-hdrs
+    configure:
+      - args: |
+          sed -e 's|_BUILD_ROOT_|@BUILD_ROOT@|' @SOURCE_ROOT@/scripts/meson-kernel-uefi-@OPTION:arch-triple@.cross-file \
+              > ./meson-custom.cross-file
```

Comment:
AFAICT this `sed` does nothing since the template does not contain `_BUILD_ROOT_`.